apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -723,7 +725,11 @@ export const renderError = ({
               )
         }
         title={<TranslatedError error={error as unknown as Error} noLink />}
-        description={<TranslatedError error={error as unknown as Error} field="description" />}
+        description={
```

Comment:
Then it might be worth making description optional in `ErrorBody`, because in `ErrorBody` if you look at `ErrorDescription` there's an extra margin added, which will create a big gap in case there is actually no description but there is 1 (or more) button.